<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Simple Video Editor</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                            <widget class="QPushButton" name="openButton">
                                <property name="text">
                                    <string>Open</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="saveButton">
                                <property name="text">
                                    <string>Save</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="cutButton">
                                <property name="text">
                                    <string>Cut</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="addTextButton">
                                <property name="text">
                                    <string>Add Text</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="combineButton">
                                <property name="text">
                                    <string>Combine</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
                    <widget class="VideoPlayerWidget" name="videoPlayerWidget" />
                </item>
                <item>
                    <widget class="TimelineWidget" name="timelineWidget" />
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>800</width>
                    <height>22</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
